ServiceMonitoringFunction/tst/com.amazon.vas.serviceordermonitoringlambda/component/GetJobMetricsComponentTest.java

```diff
@@ -0,0 +1,61 @@
+package component;
+
+import com.amazon.vas.serviceordermonitoringlambda.builder.JobDetailsBuilder;
+import com.amazon.vas.serviceordermonitoringlambda.component.GetJobMetricsComponent;
+import com.amazon.vas.serviceordermonitoringlambda.model.GetJobMetricsInputBO;
+import com.amazon.vas.serviceordermonitoringlambda.model.GetJobMetricsOutputBO;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import util.DefaultModelBuilders;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static util.TestConstants.CITY;
+import static util.TestConstants.MERCHANT_ID;
+
+public class GetJobMetricsComponentTest {
+
+    private DefaultModelBuilders defaultModelBuilders;
```

Comment:
put it below other public fields.
private DefaultModelBuilders defaultModelBuilders = new DefaultModelBuilders();